### PR TITLE
Add NetworkAttachments

### PR DESCRIPTION
**Pull request by cschwede**

### CODE REVIEW

File: api/v1beta1/swiftstorage_types.go [hunk 1]
```diff
@@ -65,19 +65,27 @@ type SwiftStorageSpec struct {
...
+	NetworkAttachments []string `json:"networkAttachments,omitempty"`
 }
 
 // SwiftStorageStatus defines the observed state of SwiftStorage
 type SwiftStorageStatus struct {
 	// ReadyCount of SwiftStorage instances
 	ReadyCount int32 `json:"readyCount,omitempty"`
 
-	// Conditions
+    // Conditions
```
Comment (fmount) [inline api/v1beta1/swiftstorage_types.go:79]: indentation

File: api/v1beta1/swiftstorage_types.go [hunk 2]
```diff
@@ -65,19 +65,27 @@ type SwiftStorageSpec struct {
...
 	Conditions condition.Conditions `json:"conditions,omitempty" optional:"true"`
+
+    // NetworkAttachments status of the deployment pods
```
Comment (fmount) [inline api/v1beta1/swiftstorage_types.go:82]: indentation

**Review by fmount — APPROVED** [review]
Other than a couple of indentation related issues, I think the patch is good to go.
@cschwede if you already tried it locally (with net isolation) and you see the interfaces being added in the Pods, feel free to move forward.

**Comment by cschwede** [comment]
Thanks @fmount, fixed the intendations in a second commit to make it clear there was no other modification.

**Pull request merged**